### ISSUE TITLE
@cycle/http probably wrong isolation function?

### ISSUE THREAD

**Issue by baranovxyz** [cyclejs/cyclejs#685]
```request._namespace.indexOf(scope) !== -1```
I mean what if I isolated some component twice with same scope name?
```
export const isolateSource = (
  source: DriverSource,
  scope: string | null
): DriverSource =>
  scope === null
    ? source
    : source.filter(
        (request: RequestOptions) =>
          Array.isArray(request._namespace) &&
          request._namespace.indexOf(scope) !== -1
      );
```

**Comment by jvanbruegge** [comment]
I created [a test](https://github.com/cyclejs/cyclejs/pull/692/files#diff-83140a82268528effa0d84ac231ef446R456) that would indeed trigger unwanted behavior without the fix.

**Comment by staltz** [comment]
Fixed by Jan, and new version cycle/http v14.4.0 released!
